### PR TITLE
More detail for user administration

### PR DESCRIPTION
**Pull request by OriPekelman**
Needs review.

### CODE REVIEW

File: using/user-administration.md [hunk 1]
```diff
@@ -91,4 +116,62 @@ or delete the user's access to the environment entirely.
...
+    sprint1: viewer
+    hot-fix: viewer
+    pr-2: viewer
+    pr-3: viewer
+Adding users can result in additional charges.
+Are you sure you want to add this user? [Y/n]
+User alice@example.com created
+```
+
+Once this has been done the user will receive an email asking her to confirm
```
Comment (ralt) [inline using/user-administration.md]: s/Once this has been done/Once this is done,/

?

Comment (ralt) [inline using/user-administration.md]: "asking her"? Shouldn't we use the passive form to avoid gender issues?

Comment (pjcdawkins) [inline using/user-administration.md]: Where a pronoun can't easily be avoided, IMO "they" / "them" is best: https://en.wikipedia.org/wiki/Singular_they

Comment (OriPekelman) [inline using/user-administration.md]: Agreed, I generally prefer they/them as neutral. Almost never "he/him" (you won't find any in the docs afaik). In the example we have Alice. I believe here she is fine.
But feel free to amend

**Pull request merged**